heartbeat/aws-vpc-move-ip

```diff
@@ -167,9 +167,10 @@ ec2ip_validate() {
 ec2ip_monitor() {
 	if ocf_is_true ${OCF_RESKEY_monapi} || [ "$__OCF_ACTION" = "start" ] || ocf_is_probe; then
 		ocf_log info "monitor: check routing table (API call)"
-		cmd="$OCF_RESKEY_awscli --profile $OCF_RESKEY_profile --output text ec2 describe-route-tables --route-table-ids $OCF_RESKEY_routing_table"
+		cmd="$OCF_RESKEY_awscli --profile $OCF_RESKEY_profile --output text ec2 describe-route-tables --route-table-ids $OCF_RESKEY_routing_table --query RouteTables[*].Routes[?DestinationCidrBlock==\`$OCF_RESKEY_address/32\`].InstanceId"
```

Comment:
The quotes here should be `'` the backslash followed by reverse quote (`).

Reply:
Oh, and I just noticed it has to end with `_ip` instead of `_address`, as the backwards-compatible to avoid issues for users using the `ip` parameter instead of the deprecated `address` parameter

Reply:
The \` cannot be replaced by `\' ` as it will break the JMSPath Query:

> Bad value for --query RouteTables[*].Routes[?DestinationCidrBlock==\'192.168.1.9/32\'].InstanceId: Bad jmespath expression: Unknown token \:
> RouteTables[*].Routes[?DestinationCidrBlock==\'192.168.1.9/32\'].InstanceId

Reply:
> Sorry, I meant ' without the backspace.

That seems to work well, I will give it a go